### PR TITLE
Minor enhancements to README.md

### PR DESCRIPTION
**Pull request by Holmistr**
Some minor enhancements that raised my eyebrows while trying to setup Nexmark benchmark.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,14 +93,14 @@ Before start to run the benchmark, you should build the Nexmark benchmark first
...
-  - Edit `flink/conf/workers` and enters the IP address of each worker node. Recommand to set 8 entries.
+  - Edit `flink/conf/workers` and enter the IP address of each worker node. Recommand to set 8 entries.
   - Replace `flink/conf/sql-client-defaults.yaml` by `nexmark/conf/sql-client-defaults.yaml`
   - Replace `flink/conf/flink-conf.yaml` by `nexmark/conf/flink-conf.yaml`. Remember to update the following configurations:
     - Set `jobmanager.rpc.address` to you master IP address
     - Set `state.checkpoints.dir` to your local file path (recommend to use SSD), e.g. `file:///home/username/checkpoint`.
     - Set `state.backend.rocksdb.localdir` to your local file path (recommend to use SSD), e.g. `/home/username/rocksdb`.
 - Step5: Configure Nexmark benchmark.
-  - Set `nexmark.metric.reporter.host` to your master IP address.
+  - Set Edit `nexmark/conf/nexmark.yaml` and set `nexmark.metric.reporter.host` to your master IP address.
```
Comment (wuchong) [inline README.md:103]: minor:
```suggestion
  - Edit `nexmark/conf/nexmark.yaml` and set `nexmark.metric.reporter.host` to your master IP address.
```

**Review by wuchong — COMMENTED** [review]
Thank you for the contribution. I left a minor comment.

**Comment by Holmistr** [comment]
@wuchong Thanks a lot for the review. Sorry for the delay, I was on vacation. Could you please give it one more round of review?

**Review by wuchong — APPROVED** [review]
Thank you for the updates.

**Pull request merged**